src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  cursor: pointer;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    color: #34568b;
+  }
+
+  &__photo {
+    background-image: url('../images/imac.jpeg');
+    width: 160px;
+    height: 134px;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
```

Comment:
It's better to use an img tag for images instead of setting them as a background. A background image will not be picked up by screen readers, making the site less accessible.